package.json

```diff
@@ -6,7 +6,13 @@
     "build": "rm -rf dist && rollup -c rollup.config.js",
     "jest": "jest",
     "jest-debug": "jest --runInBand --useStderr",
-    "serve": "cd src/examples && ../../node_modules/.bin/webpack serve"
+    "serve": "cd src/examples && ../../node_modules/.bin/webpack serve",
+    "lint": "run-p lint:*",
+    "lint:eslint": "eslint \"src/**/*.{ts,js}\"",
+    "lint:prettier": "yarn prettier --check .",
```

Comment:
@quolpr why separate `eslint` from `prettier`? Ideally `prettier` is just a plugin from `eslint` so they are executed together and there's no need to separate the commands. You can have `lint` and `lint:fix` and that's it.